platform/ios/app/MBXAnnotationView.m

```diff
@@ -0,0 +1,78 @@
+#import "MBXAnnotationView.h"
+
+static const CGFloat size = 35.0;
+static const CGFloat padding = 7.5;
```

Comment:
Nit: Globals should be named the same way as classes, in UpperCamelCase with a prefix.